quinn/src/send_stream.rs

```diff
@@ -287,6 +289,15 @@ impl SendStream {
     ) -> Poll<Result<usize, WriteError>> {
         pin!(self.get_mut().write(buf)).as_mut().poll(cx)
     }
+
+    /// Set whether the stream automatically resets rather than finishes when dropped
+    ///
+    /// If this stream is dropped before [`finish()`](Self::finish) or [`reset()`](Self::reset) is
+    /// called on it, it automatically finishes if `error_code` is set to `None`, or automatically
+    /// resets with `error_code` if `error_code` is set to `Some`. Defaults to `None`.
+    pub fn set_reset_on_drop(&mut self, error_code: Option<VarInt>) {
```

Comment:
Would it make more sense to drop the Option wrapper from the argument here?

Reply:
It seems conceivably useful that it can be both set and "unset"